README.TXT

```diff
@@ -1,3 +1,32 @@
+Ultimate Tapan Kaikki
+=====================
+
+This is the legendary 1990's game Ultimate Tapan Kaikki, ported to
+SDL2. It runs at least on macOS, Linux, FreeBSD and Windows.
+
+Suomipelit GitHub: https://github.com/suomipelit
+Suomipelit Slack: https://tinyurl.com/suomipelit-slack

```

Comment:
I think GitHub doesn't render these links as clickable because this is a txt file, which is a bit lame. Should we create a README.md file to put this new stuff in, or convert this whole file to Markdown?

Reply:
Never mind, GitHub actually renders them as links.